core/control-plane/control-plane-aggregate-services/build.gradle.kts

```diff
@@ -23,9 +23,7 @@ dependencies {
     implementation(project(":spi:common:transaction-spi"))
     implementation(project(":spi:control-plane:asset-spi"))
     implementation(project(":spi:control-plane:transfer-data-plane-spi"))
-
```

Comment:
Please do not modify files that are not relevant to the PR, particularly if it just remove lines

Reply:
done